project_categ/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Project Category
+================
+
+* This module create a parent Category (Tag). E.g. "System Type"..
```

Comment:
Add following sentence:
Adds Project specific category lists in Tasks: 
(and blank line before the list)